### PR TITLE
[zh-cn] Updates > Inside look at modern web browser (part 1)

### PR DESCRIPTION
**Pull request by kennith**
Reference: https://developers.google.com/web/updates/2018/09/inside-browser-part1

[Check out a review request at GitLocalize](https://gitlocalize.com/repo/107/zh-CN/review/8516)

### CODE REVIEW

**Comment by kennith** [comment]
@daix6, @maplelinst, and @binggg please sign the CLA if you have not done so.

**Comment by maplelinst** [comment]
I've done it now. @binggg @daix6 Would you please do it when you're available?

**Comment by daix6** [comment]
I've signed it.

**Comment by suhailsinghbains** [comment]
Hi @binggg 
Please sign the CLA and comment "I've signed it" here
Thanks

**Comment by suhailsinghbains** [comment]
Hi @binggg 
Please sign the CLA and comment "I've signed it" here
Thanks

**Comment by binggg** [comment]
I've signed it.

File: src/content/zh-cn/updates/2018/09/inside-browser-part1.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+  }
+</style>
+
+# 深入探索现代网络浏览器（第1部分） {: .page-title }
+
+{% include "web/_shared/contributors/kosamari.html" %}
+
+## CPU、GPU、内存和多进程架构
+
+在这个4 部分的系列博客文章中，我们将介绍 Chrome 浏览器从高级架构到渲染流程的具体细节。如果你想知道浏览器如何将你的代码转换为一个具备功能的网站，或者你不确定为什么某种特定的技术被建议用来提高性能，那么本系列非常适合你。
```
Comment (binggg) [inline src/content/zh-cn/updates/2018/09/inside-browser-part1.md]: Should there be a space before 4

File: src/content/zh-cn/updates/2018/09/inside-browser-part1.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+
+作为本系列的第1部分，我们将介绍核心计算术语和 Chrome 的多进程架构。
```
Comment (binggg) [inline src/content/zh-cn/updates/2018/09/inside-browser-part1.md]: done

File: src/content/zh-cn/updates/2018/09/inside-browser-part1.md [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+
+## 反馈 {: .hide-from-toc }
+
+{% include "web/_shared/helpful.html" %}
+
+<div class="clearfix"></div>
+
+{% include "web/_shared/rss-widget-updates.html" %}
+
+{% include "web/_shared/rss-widget-updates.html" %}
```
Comment (binggg) [inline src/content/zh-cn/updates/2018/09/inside-browser-part1.md]: done

**Comment by ilyaspiridonov** [comment]
Translation reviewed @agektmr

**Pull request closed without merging**